requirements.txt

```diff
@@ -6,3 +6,4 @@ jsonschema~=3.2.0
 parsedatetime~=2.6
 pytz~=2020.1
 python-dateutil~=2.8.1
+PyYAML~=5.3.1
```

Comment:
5.4.1 is current. Also, consider bumping other dependencies every once in a while.